<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mirall::AccountSettings</class>
 <widget class="QWidget" name="Mirall::AccountSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QGroupBox" name="maintenanceGroupBox">
     <property name="title">
      <string>Account Maintenance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="ignoredFilesButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Edit Ignored Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="modifyAccountButton">
        <property name="text">
         <string>Modify Account</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="syncStatusGroupBox">
     <property name="title">
      <string>Account to Synchronize</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="connectLabel">
        <property name="text">
         <string>Connected with &lt;server&gt; as &lt;user&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="_folderList"/>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="_buttonAdd">
          <property name="text">
           <string>Add Folder...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonEnable">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="storageGroupBox">
     <property name="title">
      <string>Storage Usage</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="quotaProgressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="quotaLabel">
        <property name="text">
         <string>Retrieving usage information...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="quotaInfoLabel">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; Some folders, including network mounted or shared folders, might have different limits.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
